follower-read.md

```diff
@@ -24,7 +24,7 @@ To enable TiDB's Follower Read feature, set the value of the `tidb_replica_read`
 {{< copyable "sql" >}}
```

Comment:
Please modify [line 22](https://github.com/pingcap/docs/pull/3122/files#diff-a2c19cd7e1403c5a9e29745e800b4b59R22): 

To enable TiDB's Follower Read feature, modify the value of the `tidb_replica_read` session variable:

Reply:
Done. PTAL.